<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="CaesarDispTips">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>位移:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CaesarDispBox">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="CaesarStepTips">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>Step:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CaesarStepBox">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="CaesarLimitTips">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>Lim:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="CaesarLimitBox">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>45</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CaesarDigitCheckBox">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>数字</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CaesarEncryptButton">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>加密</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="CaesarTextBox"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CaesarDecryptButton">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>45</height>
          </size>
         </property>
         <property name="text">
          <string>解密</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="CaesarCipherBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
